File: 3195-separate-black-and-white-balls/separate-black-and-white-balls.js
/**
 * @param {string} s
 * @return {number}
 */
var minimumSteps = function(s) {
    let n = s.length
    let count = 0
    let zero = 0
    for (let i = n-1;i>-1;i--){
        if (s[i]==="1"){ count += zero}
        else { zero++ }
    }
    return count
};

File: 3195-separate-black-and-white-balls/separate-black-and-white-balls.py
class Solution:
    def minimumSteps(self, s: str) -> int:

        n = len(s)
        count = 0
        zero = 0
        for i in range(n-1,-1,-1):
            if s[i]=="1":
                count+=zero
            else:
                zero+=1

        return count